src/pybind/mgr/dashboard/module.py

```diff
@@ -91,7 +91,7 @@ class ServerConfigException(Exception):
     pass
 
 
-class SSLCherryPyConfig(object):
```

Comment:
@b-ranto 
could you prefix the title of your commit message with the subcomponent your are changing ? in this case, it would be "mgr/dashboard:" . thanks!